packages/selleo-design-core/src/Colors.tsx

```diff
@@ -0,0 +1,217 @@
+      },
+    ],
+    secondary: [
+      {
+        title: 'secondary.500',
+        description: 'HEX #5C99D2 | hsl(209, 57, 59) | Pantone 298 C',
+        class: 'bg-brand-secondary-500 [&_p]:text-white [&_h4]:text-white',
+      },
+      {
+        title: 'Secondary.600',
+        description: 'HEX #3871A5 | hsl(209, 49, 43)',
+        class: 'bg-brand-secondary-600 [&_p]:text-white [&_h4]:text-white',
+      },
+      {
+        title: 'secondary.200',
+        description: 'HEX #DFEBF6 | hsl(209, 56, 92)',
+        class:
+          'bg-brand-secondary-200 [&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black',
+      },
+      {
+        title: 'secondary.100',
+        description: 'HEX #F3F8FC | hsl(207, 60, 97)',
+        class:
+          'bg-brand-secondary-100 [&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black',
+      },
+    ],
+  },
+  neutral: [
+    {
+      title: 'black',
+      description: 'HEX #2F3033 | hsl(225, 4, 19) | Pantone 426 C',
+      class: 'bg-black [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'neutral.600',
+      description: 'HEX #4C4D52 | hsl(230, 4, 31)',
+      class: 'bg-neutral-600 [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'neutral.500',
+      description: 'HEX #696B72 | hsl(227, 4, 43)',
+      class: 'bg-neutral-500 [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'neutral.400',
+      description: 'HEX #95979D | hsl(225, 4, 60)',
+      class: 'bg-neutral-400 [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'neutral.300',
+      description: 'HEX #BFC0C4 | hsl(228, 4, 76)',
+      class: 'bg-neutral-300 [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'neutral.200',
+      description: 'HEX #EAEAEB | hsl(240, 2, 92)',
+      class:
+        'bg-neutral-200 [&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black mb-3',
+    },
+    {
+      title: 'neutral.100',
+      description: 'HEX #F4F5F5 | hsl(180, 5, 96)',
+      class:
+        'bg-neutral-100 [&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black mb-3',
+    },
+    {
+      title: 'white',
+      description: 'HEX #FFFFFF | hsl(0, 0, 100)',
+      class:
+        'bg-white [&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black border-[1px] border-neutral-200 mb-3',
+    },
+    {
+      title: 'hero.overlay',
+      description: 'HEX #2F3033 | hsl(225, 4, 19) 70%',
+      class: 'bg-neutral-hero-overlay [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+  ],
+  states: [
+    {
+      title: 'danger.500',
+      description: 'HEX #E81B2F | hsl(354, 82, 51)',
+      class: 'bg-danger [&_p]:text-white [&_h4]:text-white mb-3',
+    },
+    {
+      title: 'transparent',
+      description: 'HEX #000000 | hsl(0, 0, 0)  0%',
+      class:
+        '[&_p]:text-black [&_h4]:text-black dark:[&_p]:text-black border-[1px] border-neutral-200 mb-3',
```

Comment:
please check if removing the text colors will fix the not visible text
![image](https://user-images.githubusercontent.com/32620457/224961146-8ada51f6-0e1f-42d2-83cc-92819186f0e2.png)

Reply:
Okay, done